src/common.h

```diff
@@ -760,6 +760,13 @@ enum
 
 	SFE_OPUS_BAD_SAMPLERATE,
 
+	SFE_WAVPACK_NEW_DECODER,
+	SFE_WAVPACK_NEW_ENCODER,
+	SFE_WAVPACK_WRITE_HEADER,
+	SFE_WAVPACK_PACK_SAMPLES,
+	SFE_WAVPACK_UNPACK_SAMPLES,
+	SFE_WAVPACK_DEAD,
```

Comment:
This error type is unnecessary. Looking at the code, this guards `psf->codec_data`, but `psf->codec_data` is only assigned twice, once at allocation, and once at close.

If you hit `psf->codec_data  == NULL` after a successful open it is allowed to return `SFE_INTERNAL`.

Also, the name is a bit out of the usual for this project :)